packages/twenty-server/src/engine/core-modules/billing/services/billing-plan.service.ts

```diff
@@ -0,0 +1,92 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+
+import { Repository } from 'typeorm';
+
+import { ProductPriceEntity } from 'src/engine/core-modules/billing/dto/product-price.entity';
+import { BillingPrice } from 'src/engine/core-modules/billing/entities/billing-price.entity';
+import { BillingProduct } from 'src/engine/core-modules/billing/entities/billing-product.entity';
+import { BillingPlanKey } from 'src/engine/core-modules/billing/enums/billing-plan-key.enum';
+import { BillingUsageType } from 'src/engine/core-modules/billing/enums/billing-usage-type.enum';
+@Injectable()
+export class BillingPlanService {
+  protected readonly logger = new Logger(BillingPlanService.name);
+  constructor(
+    @InjectRepository(BillingPrice, 'core')
+    private readonly billingPriceRepository: Repository<BillingPrice>,
+    @InjectRepository(BillingProduct, 'core')
+    private readonly billingProductRepository: Repository<BillingProduct>,
+  ) {}
+
+  async getProductByPlan(
```

Comment:
getBaseProductFStripeIdorPlan ? is is still useful
or getProductStripeIdsForPlan  ? is is still useful